### ISSUE TITLE
Permitir doações via pagseguro

### ISSUE THREAD

**Issue by lcobucci** [PHPSC/phpsc-conf#43]
Também possui dependência da #42, porém é algo bem importante.
Minha ideia é criarmos a possibilidade das pessoas realizarem doações como pessoas físicas ou como pessoas jurídicas, identificadas através do CPF ou CNPJ.

Temos que ver a questão de valores mínimos (minhas sugestão é uns R$ 30,00).
@mingomax pode cadastrar aqui os detalhes do documento que deve ser gerado?

@djhonyy, @kassner, @wcomnisky, @marcelaraujo, @mrprompt sugestões?

**Comment by djhonyy** [comment]
Eu permitiria qualquer valor, a partir dos 0,50 mínimos que o PagSeguro exige. Doação é sempre bem vinda, e o que importa é a intenção de ajudar.

Podemos, se for o caso criarmos uma listagem das maiores doações. O que acham?

**Comment by lcobucci** [comment]
Acho bacana @djhonyy :smile:

**Comment by marcelaraujo** [comment]
E se a pessoa quiser doar de forma anônima?

Eu já fiz doações ao projeto do Eclipse por exemplo e, foi de forma anônima.

**Comment by djhonyy** [comment]
Enquadramos em anônimos.

**Comment by lcobucci** [comment]
Imagino que o único detalhe da doação anônima é a divulgação, ou seja só um filtro na lista de doadores né?

**Comment by djhonyy** [comment]
Exatamente, quem quiser doar e aceitar a divulgação blz, mas quem quiser ficar na moita, vai pra lista de doações a anônimas.

**Comment by lcobucci** [comment]
As doações de PJ podem usar a entidade Empresa descrita na tarefa #46

**Comment by wcomnisky** [comment]
Precisamos nos informar a respeito de questões legais para doações a partir de empresas. Como não possuímos CNPJ, acredito que não seja possível.
